app/views/details/index.njk

```diff
@@ -96,16 +96,24 @@
       classes: 'mpdp-margin-top-negative-3',
       summaryHtml: '<div class="govuk-!-font-size-16">Report a problem</div>',
       id: 'reportProblem',
-      html: "<p>If you're the agreement holder for this business, you can contact the Rural Payments Agency.</p>
-
+      html: "
       <div>
+      <b>Rural Payments Agency</b><br>
       Email: <a href='mailto:ruralpayments@defra.gov.uk'>ruralpayments@defra.gov.uk</a><br>
       Telephone: 03000 200 301<br>
-      Monday to Friday, 9am to 5pm<br>
-      <a id='callCharges' target='_blank' href='https://www.gov.uk/call-charges'>Find out more about call charges</a>
-      </div>
+      Monday to Friday, 8.30am to 5pm<br>
+
       <br>
-      <p>Or you can <a id='sfiQueryForm' target='_blank' href='https://www.gov.uk/government/publications/sustainable-farming-incentive-pilot-query-form'>use the Sustainable Farming Incentive pilot query form</a></p>"
+      <p>
+        <b>Sustainable farming incentive pilot</b><br>
```

Comment:
Based on the screenshot first letter of each word should be capitalised here.

Reply:
agreed, updated accordingly